README.md

```diff
@@ -3,12 +3,35 @@
 # Xlang
 
 ## Overview
-Xlang is a system that enables a component written in any supported language to be callable from any other supported language using object-oriented programming constructs.  
+The xlang project enables developers to take existing shared libraries,
+implemented in one programming language,
+and make that library's APIs available to client code using a different programming language.
+Thus the name "xlang," for cross-language.
+
+Additionally, all of the xlang tooling will be platform-agnostic therefore will work on your favorite operating system.
+This means that if your shared library is platform-agnostic (i.e. portable to various operating systems),
+then you can use the xlang tooling to make that shared library available to various client programming language on those various platforms.
+
+A *hypothetical* example might be taking the Python numpy library and making it available to C# applications running on Linux.
+
+We're a *long* way was achieving that ultimate goal. We are in the very, very early stages of starting the project.
```

Comment:
Avoid first person. "This project is under active development, and is not complete. No release date is set for the initial release at this time.